components/analytics/usage-dashboard.tsx

```
@@ -0,0 +1,305 @@
+"use client"
+
+import { DateRangeFilter } from "@/components/filters/date-range-filter"
+import { Button } from "@/components/ui/button"
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card"
+import { ChartContainer, ChartTooltip, ChartTooltipContent, recharts } from "@/components/ui/chart"
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs"
+import { fetchTokenConsumption, fetchUserTokens } from "@/lib/api"
+import { chartColors } from "@/lib/chart-colors"
+import { DailyTokenConsumption, UserTokensResponse } from "@/lib/types"
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)